modules/drivers/snowflake/src/metabase/driver/snowflake.clj

```diff
@@ -199,10 +200,6 @@
 (defmethod sql.qp/unix-timestamp->honeysql [:snowflake :milliseconds] [_ _ expr] [:to_timestamp expr 3])
 (defmethod sql.qp/unix-timestamp->honeysql [:snowflake :microseconds] [_ _ expr] [:to_timestamp expr 6])
 
-(defmethod sql.qp/current-datetime-honeysql-form :snowflake
```

Comment:
This was defined twice in the file. Another definition can be found on [this line](https://github.com/metabase/metabase/blob/3f1c95ad3f7c733f93629c1730a1707845478d52/modules/drivers/snowflake/src/metabase/driver/snowflake.clj#L547)